opentelemetry-api/src/opentelemetry/_events/__init__.py

```diff
@@ -0,0 +1,196 @@
+from abc import ABC, abstractmethod
+from logging import getLogger
+from os import environ
+from typing import Any, Optional, cast
+
+from opentelemetry._logs import LogRecord
+from opentelemetry._logs.severity import SeverityNumber
+from opentelemetry.environment_variables import (
+    _OTEL_PYTHON_EVENT_LOGGER_PROVIDER,
+)
+from opentelemetry.trace.span import TraceFlags
+from opentelemetry.util._once import Once
+from opentelemetry.util._providers import _load_provider
+from opentelemetry.util.types import Attributes
+
+_logger = getLogger(__name__)
+
+
+class Event(LogRecord):
+
+    def __init__(
+        self,
+        name: str,
+        timestamp: Optional[int] = None,
+        trace_id: Optional[int] = None,
+        span_id: Optional[int] = None,
+        trace_flags: Optional["TraceFlags"] = None,
+        payload: Optional[Any] = None,
```

Comment:
This is called body both in the specs and in the LogRecord, why using a different name?

Reply:
I was trying to follow the semantic conventions: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/events.md where it said "In addition to a required name, an Event may contain a payload (data) of any type permitted by the [LogRecord body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body). In its implementation, the Event payload (data) will constitute the Body of the LogRecord. Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe the event context.".

Changed it to `body` in the new commit.